### PR TITLE
Create estimate ind cqc filled posts - early PR for comments

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
Early PR for Gary to leave comments on

# How to test

# Developer checklist
- [ ] Unit test
- [ ] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/estimate_ind_cqc_filled_posts.py [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    PartitionKeys as Keys,
+    IndCqcColumns as IndCqc,
+)
+from utils.estimate_job_count.models.care_homes import model_care_homes
+from utils.estimate_job_count.models.primary_service_rolling_average import (
+    model_primary_service_rolling_average,
+)
+from utils.estimate_job_count.models.extrapolation import model_extrapolation
+from utils.estimate_job_count.models.interpolation import model_interpolation
+from utils.estimate_job_count.models.non_res_with_pir import (
```
Comment (garypolzin) [inline jobs/estimate_ind_cqc_filled_posts.py:23]: rename folder from estimate_job_count to estimate_filled_posts

File: jobs/estimate_ind_cqc_filled_posts.py [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+            (
+                F.col(IndCqc.estimate_filled_posts).isNull()
+                & (F.col(IndCqc.ascwds_filled_posts_dedup_clean).isNotNull())
+            ),
+            F.col(IndCqc.ascwds_filled_posts_dedup_clean),
+        ).otherwise(F.col(IndCqc.estimate_filled_posts)),
+    )
+
+    df = update_dataframe_with_identifying_rule(
+        df, "ascwds_job_count", IndCqc.estimate_filled_posts
```
Comment (garypolzin) [inline jobs/estimate_ind_cqc_filled_posts.py:186]: change to `ascwds_filled_posts`

**Review by garypolzin — COMMENTED** [review]
just going to put a generic message here to say to try to get rid of all references to job_count!
columns look to have all been converted but functions and folder names still refer to it

File: jobs/estimate_ind_cqc_filled_posts.py [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    IndCqc.primary_service_type,
+    IndCqc.people_directly_employed,
+    IndCqc.number_of_beds,
+    IndCqc.cqc_location_import_date,
+    IndCqc.ascwds_filled_posts,
+    IndCqc.ascwds_filled_posts_source,
+    IndCqc.ascwds_filled_posts_dedup_clean,
+    IndCqc.current_cssr,
+    IndCqc.cqc_sector,
+]
```
Comment (garypolzin) [inline jobs/estimate_ind_cqc_filled_posts.py:44]: might be worth bringing in current region too, just thinking of columns we might want to include in any tableau dashboards on this dataset

File: jobs/estimate_ind_cqc_filled_posts.py (hunk 1 above)
Comment (MarionJHolloway) [reply jobs/estimate_ind_cqc_filled_posts.py:23]: done

File: jobs/estimate_ind_cqc_filled_posts.py (hunk 2 above)
Comment (MarionJHolloway) [reply jobs/estimate_ind_cqc_filled_posts.py:186]: done

File: jobs/estimate_ind_cqc_filled_posts.py (hunk 3 above)
Comment (MarionJHolloway) [reply jobs/estimate_ind_cqc_filled_posts.py:45]: done

**Pull request closed without merging**